CMakeLists.txt

```diff
@@ -1025,7 +1031,6 @@ add_custom_target(check
   DEPENDS ${STUMPLESS_FUNCTION_TESTS}
 )
 
-
```

Comment:
don't remove this extra newline which acts as a section separator

Reply:
I've re-added this newline.